examples/launch_tensorboard_example.py

```diff
@@ -21,16 +21,11 @@
 from pathlib import Path
```

Comment:
For each of the deleted example, can you grep the codebase to make sure they dont appear in the doc or faq?